src/main/java/com/nylas/NeuralOcr.java

```diff
@@ -0,0 +1,32 @@
+package com.nylas;
+
+import java.util.List;
+
+public class NeuralOcr extends File {
+	private List<String> ocr;
+	private int processed_pages;
```

Comment:
this field is listed in the schema, but not the example.  do you see it in testing?

Reply:
Yup, the documentation is outdated unfortunately, here's a live response:

```
{
    "account_id": "id",
    "content_type": "application/pdf",
    "filename": "sample.pdf",
    "id": "id",
    "object": "file",
    "ocr": [
        "page 1",
        "page 2"
    ],
    "processed_pages": 2,
    "size": 433994
}
```

I removed some of the IDs in the response